test/Driver/autolink-force-load-no-comdat.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: not %swiftc_driver -incremental -autolink-force-load %s 2>&1 | %FileCheck -check-prefix=AUTOLINK_FORCE_LOAD %s
+// RUN: not %swiftc_driver -autolink-force-load -incremental %s 2>&1 | %FileCheck -check-prefix=AUTOLINK_FORCE_LOAD %s
```

Comment:
There's no real reason to put this in a separate test file; since you're just testing the driver behavior, you can use `-###` and an explicit triple to check for the error cases.

Reply:
Hmm, yes, this is driver, not IRGen.  My mistake.  Either way, #26635 will remove the file.